### PR TITLE
Fix loaded nulls

### PR DESCRIPTION
**Pull request by pacejackson**
Currently, when you load a model from the database, it ends up setting `explicit = True` on all of the values which means that `model._values['column'].deleted` resolves to `True` when you load null values from the database.  As a result, cqlmapper issues a `DELETE` query for all of the null values that you loaded, creating a tombstone for each one when you don't need one.

This PR ports two updates from cqlengine that address this issue:

https://github.com/datastax/python-driver/commit/589c15abeee37667824c270d8fc3db69912f63e8
https://github.com/datastax/python-driver/commit/99cc8e83a0edc662808b509ec328d889b85c72c5

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ Vagrant.configure(2) do |config|
   config.vm.network :forwarded_port, guest: 9090, host: 9090
 
   config.vm.provider :virtualbox do |vb|
-    vb.customize ["modifyvm", :id, "--memory", "2048"]
+    vb.customize ["modifyvm", :id, "--memory", "4096"]
```
Comment (fishy) [inline Vagrantfile]: This change seems unrelated to the issue?

Comment (pacejackson) [reply Vagrantfile]: It is!  That's why it's its own commit :)

**Comment by pacejackson** [comment]
Travis shouldn't even be running...
